files/weekly-summary.py

```diff
@@ -212,22 +224,22 @@ def summarize_github_prs(what, body_json):
                     happened = datetime.strptime(pr[when], gh_time_format)
                     # If this pull request was created outside of the summary
                     # period, skip it.
-                    if happened.date() < start_day or happened.date() > end_day:
+                    if end_day < happened.date() < start_day:
```

Comment:
Thanks for the catch. I missed the change from `or` to `and` when flipping this around.